### PR TITLE
Governance Template UI

### LINKED ISSUE #1840: [TASK] Simple governance template UI
# Overview
A simple UI for quickly iterating on ways to interact with a governance system, with different form inputs for submitting governance related transactions.

Image

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes

_Provide a detailed description of proposed changes._

- Governance Template UI component in `webb-ui-components` lib

### Proposed area of change

_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [ ] `apps/tangle-dapp`
- [ ] `apps/faucet`
- [x] `libs/webb-ui-components`

### Reference issue to close (if applicable)

_Specify any issues that can be closed from these changes (e.g. Closes #233)._

- Closes #1840 

### Screen Recording

_If possible provide a screen recording of proposed change._

https://github.com/webb-tools/webb-dapp/assets/69841784/640b1415-d6df-46f2-8fc0-a97bdc033e41

---

### Code Checklist

_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by drewstone — APPROVED** [review]
Awesome work @vutuanlinh2k2

File: libs/webb-ui-components/src/components/GovernanceForm/SelectChainRing.tsx
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/GovernanceForm/SelectChainRing.tsx:1]: Please avoid duplicating the component. We already have this component present within the confirmation card, so why is there a need to reimplement it here? All we need to do is add a callback `onItemClick` to the component and invoke that function whenever users interact with it.

While duplicating code might seem quicker initially, it complicates our maintenance efforts and makes future changes more challenging.

File: libs/webb-ui-components/src/stories/templates/GovernanceForm.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,973 @@
...
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+export const Default: Story = {
+  render: () => (
+    <GovernanceForm
+      governanceFncNames={['setHandler', 'setVerifier']}
+      typedChainIdSelections={Object.keys(chainsConfig)
+        .slice(0, 8)
+        .map((chainId) => +chainId)}
+      abi={[
```
Comment (AtelyPham) [inline libs/webb-ui-components/src/stories/templates/GovernanceForm.stories.tsx:23]: Please use `VAnchor__factory.abi`; there's no need to import the entire ABI in this instance.

**Review by AtelyPham — COMMENTED** [review]
Left some comments.

**Comment by drewstone** [comment]
@vutuanlinh2k2 it would be good to create a page in our dApp called "Governance".

Ideally it has dropdowns:
- Select the bridge
- Select the contract
- See all governance functions for that contract
- Be able to submit the proposal transactions

Ideally there's a little component where we list all contracts with respect to this bridge hyperlinked to their addresses on a block explorer. It can change depending on what chain is actively selected in the diagram. It can even show some metadata about the governance settings of the bridge/each side of the bridge, some of the data we have on the stats dApp for example (what tokens are wrappable on the active side, what the fee percentage is)

And to make it even more substantial, we can get more detailed:
- For inputs that require a token (addToken, removeToken) we can replace the basic input with a token selector.

This would be a good checklist to make.

**Review by AtelyPham — APPROVED** [review]

**Pull request merged**